### DISCUSSION TITLE
2024-01-25 CUE Modules Feedback Session / office hours

### DISCUSSION THREAD

**Discussion by dominikdm** [cue-lang/cue#2777] [Community Calls]
The next Modules Feedback Session will be on Thursday, Jan 25, 2024 at 1600 UTC.

See this discussion for context on the modules proposal: https://github.com/cue-lang/cue/discussions/2330

Please reply with any questions/topics you would like to see covered.

### Rough agenda
- improved end-to-end tests
- plans for generated code

### Meeting details
https://meet.google.com/wis-hrtw-isf

This session will be recorded for those unable to join us live.

### CUE Community shared calendar
[Subscribe to the shared calendar](https://calendar.google.com/calendar/u/0?cid=Y19lNzkxMWQ5OWQ4ZGIyMmU2ZTVjMzhkMTVkNjY2ZTVlNjdiNWE5ODNkZWU4N2JmNTU2NDY3NzI1OGIxYjJhMTFhQGdyb3VwLmNhbGVuZGFyLmdvb2dsZS5jb20).

### Recording
https://youtu.be/HtjAT3gx4cg

### Slides/materials
https://docs.google.com/presentation/d/1ETjAPvou_foI_h6wiyNw7GKQnIEuywHwl8zeivsNEEA/edit?usp=sharing

### Auto-generated transcript
https://docs.google.com/document/d/1gij846SHE_vbq4UT-ZlSwcsbLScSXtvYs-ax9pytzrA/edit?usp=sharing

**Comment by mvdan** [comment]
We had to move due to technical issues with the recording: https://meet.google.com/dhk-vzpm-fej

**Comment by dominikdm** [comment]
Added recording, slides and transcript.
